docs/src/main/sphinx/release/release-456.md

```diff
@@ -0,0 +1,64 @@
+# Release 456 (4 Sep 2024)
+
+## General
+
+* Expose additional JMX metrics about resource groups, including CPU and memory
+  usage, limits, and scheduling policy. ({issue}`22957`)
+* Add [adaptive plan optimization](/optimizer/adaptive-plan-optimizations) which
+  can reorder joins at runtime. The feature can be disabled with the
+  `fault-tolerant-execution-adaptive-join-reordering-enabled` configuration
+  property or the `fault_tolerant_execution_adaptive_join_reordering_enabled`
+  session property. ({issue}`23046`)
+
+## JDBC driver
+
+* Add publishing a JDBC driver JAR without shaded, third-party dependencies. ({issue}`22098`)
+
+## BigQuery connector
+
+* Fix failures with queries using table functions when `parent-project-id` is
+  defined. ({issue}`23041`)
+
+## Blackhole connector
+
+* Add support for replacing tables. ({issue}`23004`)
+
+## Delta Lake connector
+
+* Add support for creating tables with deletion vectors. ({issue}`22104`)
+* Add support for lock-less Delta Lake write reconciliation on S3. ({issue}`23145`)
```

Comment:
What’s the user visible impact of this? “Lockless” and “write reconciliation” are implementation-level concepts.

Reply:
Reworded and pushed

Reply:
I still wonder whether we need to mention lockless and write reconciliation. Are those well-known terms in the Delta Lake spec or just details of our implementation?